lib/index.js

```diff
@@ -923,12 +926,16 @@ class Parser extends Transform {
   }
   __isDelimiter(chr, buf, pos){
     const {delimiter} = this.options
-    const delLength = delimiter.length
-    if(delimiter[0] !== chr) return 0
-    for(let i = 1; i < delLength; i++){
-      if(delimiter[i] !== buf[pos+i]) return 0
+    for(let i = 0; i < delimiter.length; i++){
+      const d = delimiter[i];
+      if(d[0] === chr){
+        for(let j = 1; j < d.length; j++){
+          if(d[j] !== buf[pos+j]) break
+        }
+        return d.length
```

Comment:
Are you sure this is right ? My reading is that if the char match the first character of the delimiter, then the loop stop and it return the delimiter length, even if the other characters don't match the ones of the delimiter? I could be wrong, just reading the code and trying to understand it.

Reply:
You are right, I made this silly mistake. I'll fix it asap.